src/modules/db/mongodb-init.ts

```diff
@@ -6,13 +6,13 @@ import * as log from '../../utils/log';
 // mongoose.set('useUnifiedTopology', true);
 // mongoose.set('useCreateIndex', true);
 
-const prepareConnectionString = (config: {
+export const prepareConnectionString = (config: {
   user: string;
   password: string;
   server: string;
   database: string;
 }) => {
-  let connectionString = 'mongodb://';
+  let connectionString: string = 'mongodb://';
```

Comment:
Do we need an explicit type here?

Reply:
Probably not. I was playing around with some things and that change must've snuck in there. Is there any harm in it?

Reply:
Shouldn't hurt I don't think, just extraneous. We should probably have a style guide for this project to tell us what to do in these situations.